cfn-templates/cid-cfn.yml

```diff
@@ -143,17 +146,22 @@ Parameters:
   CidVersion:
     Type: String
     MinLength: 5
-    Default: 0.2.30
+    Default: 0.2.31
     Description: A version of CID package
   Suffix:
     Type: String
     Description: Leave Empty. Do not use this Suffix it is not fully supported. For testing purposes only.
     Default: ""
   DeployCUDOSDashboard:
     Type: String
-    Description: Deploy CUDOS Dashboard
+    Description: Set to 'no' to remove deprecated (v4) version of CUDOS Dashboard
```

Comment:
How about forcing the user to choose either, so they don't have to set his to know. e.g. force it with AllowedValues or maybe somehow else ... point being if user selects V5 we should remove V4 maybe or something like this.

Reply:
The idea is to allow customer a double run